modules/wildfly/pom.xml

```diff
@@ -95,13 +95,12 @@
             <artifactId>hibernate-envers</artifactId>
             <optional>true</optional>
         </dependency>
-        <!-- This is to satisfy Neo4J's custom Lucene,
-          not to be confused with the one used by Hibernate Search -->
+        <!-- This is to satisfy Neo4J's custom Lucene: might not match the needs of Hibernate Search -->
```

Comment:
Yes the point is that each of these would otherwise bring their own
transitive dependency. There are minor mismatches, like in this case a
micro version although we can manage some other combinations too. The only
reasonable strategy is to manage this explicitly ourselves rather than who
comes first transient..

On Wed, 10 Aug 2016, 09:50 Gunnar Morling, <notifications@github.com> wrote:

> In modules/wildfly/pom.xml
> <https://github.com/hibernate/hibernate-ogm/pull/744#discussion_r74207596>
> :
>
> > @@ -95,13 +95,12 @@
> >              <artifactId>hibernate-envers</artifactId>
> >              <optional>true</optional>
> >          </dependency>
> > -        <!-- This is to satisfy Neo4J's custom Lucene,
> > -          not to be confused with the one used by Hibernate Search -->
> > +        <!-- This is to satisfy Neo4J's custom Lucene: might not match the needs of Hibernate Search -->
>
> Is this comment actually true any longer, now that it's only one version?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hibernate/hibernate-ogm/pull/744/files/71a373d2a2508724790c0ef12def67df85162a65#r74207596>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACmTRoBUR8r0RFfJCclziSpAbJzANDpks5qeZDlgaJpZM4JgHE1>
> .
>